go-bindings/threshold.go

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) 2021 The Dash Core developers
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//    http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package blschia
+
+// #cgo LDFLAGS: -lbls-dash
+// #cgo CXXFLAGS: -std=c++14
+// #include <stdlib.h>
+// #include "threshold.h"
+// #include "blschia.h"
+import "C"
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+	"runtime"
+	"unsafe"
+)
+
+// HashSize is the allowed size of a hash
+const HashSize = 32
```

Comment:
Do you mean `sha256.Size`? If yes, this const is not needed

Reply:
in fact, it should be `sha256`,
I would rather declare my own exportable constant for this